### ISSUE TITLE
Make ssh the default behavior for exec and port-forward

### ISSUE THREAD

**Issue by pchico83** [okteto/okteto#1076]
**Is your feature request related to a problem? Please describe.**
SSH tunnels are more stable than `kubectl port-forward`

**Additional context**
If `~/.ssh/config` exists with `rw-------` permission, `okteto up` fails:
```
$ okteto up            
 x  couldn't forward traffic to your development container: failed to add entry to your SSH config file
```

Also, if the local port needs admin access, we are showing a cryptic error:
```
$ okteto up 
 x  couldn't create pid file for pchico83 - proxy
```

**Comment by rberrelleza** [comment]
@pchico83 I can't reproduce the issue with the permissions, can you check if 600 was the permissions you had?

**Comment by rberrelleza** [comment]
@pchico83 which ports require "admin access"? I tried with port 80, but didn't got any errors.

**Comment by pchico83** [comment]
@rberrelleza never mind. Both errors are related with an initial execution with `sudo okteto up`.

**Comment by pchico83** [comment]
For port 80, `okteto up` siletly fails:
```
time="2020-10-07T05:01:23+02:00" level=info msg="ssh forward localhost:80->0.0.0.0:9900 -> failed to listen on local address: listen tcp 127.0.0.1:80: bind: permission denied" action=fe8decb5-34e8-4821-a79f-5bf1841c3c9b version=64ddea1a
```
Therefore, the port forward is not working, although `okteto up` does not complain.

**Comment by pchico83** [comment]
Closed by #1151
